 This patch fixes a bootstrap creation file problem when
 all daemons have the same name. It fixes bug #1233.

 Apply it to trunk with:

 cd <bacula-source>
 patch -p0 < fix_1233_bootstrap_name.patch
 ./configure <your options>
 make
 ...
 make install



Index: src/filed/job.c
===================================================================
--- src/filed/job.c	(révision 8483)
+++ src/filed/job.c	(copie de travail)
@@ -1190,7 +1190,7 @@
    free_bootstrap(jcr);
    P(bsr_mutex);
    bsr_uniq++;
-   Mmsg(fname, "%s/%s.%s.%d.bootstrap", me->working_directory, me->hdr.name,
+   Mmsg(fname, "%s/%s.%s.%d.fd.bootstrap", me->working_directory, me->hdr.name,
       jcr->Job, bsr_uniq);
    V(bsr_mutex);
    Dmsg1(400, "bootstrap=%s\n", fname);
Index: src/stored/fd_cmds.c
===================================================================
--- src/stored/fd_cmds.c	(révision 8483)
+++ src/stored/fd_cmds.c	(copie de travail)
@@ -344,7 +344,7 @@
    }
    P(bsr_mutex);
    bsr_uniq++;
-   Mmsg(fname, "%s/%s.%s.%d.bootstrap", me->working_directory, me->hdr.name,
+   Mmsg(fname, "%s/%s.%s.%d.sd.bootstrap", me->working_directory, me->hdr.name,
       jcr->Job, bsr_uniq);
    V(bsr_mutex);
    Dmsg1(400, "bootstrap=%s\n", fname);
